src/widgets/hits/hits.tsx

```diff
@@ -36,65 +51,83 @@ const renderer = ({ renderState, cssClasses, containerNode, templates }) => (
       hits={receivedHits}
       results={results}
       templateProps={renderState.templateProps}
-      insights={insights}
+      insights={insights as InsightsClientWrapper}
```

Comment:
how come this isn't a given, since it's a parameter?

Reply:
It's not important ( as is something internal ). What happens here is: The given `insights` may be undefined. But in reality, is never undefined as the connector always give us a insights instance for the widget. WDYT?